### PR TITLE
Apply feedback from consensus call

### PR DESCRIPTION
**Pull request by OR13**
This pull request addressed comments on the consensus call conducted on the list:

https://mailarchive.ietf.org/arch/msg/spice/TTgOt6qI3CzILzV4i34nLmkmeMc/

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]
Good improvements, thank you.

File: charter.md [hunk 1]
```diff
@@ -32,18 +28,29 @@ The SPICE WG will profile existing IETF technologies and address residual gaps t
 
 The SPICE WG, coordinates with RATS, OAuth, JOSE, COSE and SCITT working groups that develop documents related to the identity and credential space. The SPICE WG builds on cryptographic primitives defined in the CFRG (e.g., BBS Signatures) and does not define novel cryptographic schemes.
 
-The SPICE WG will not develop digital credentials for any particular use case.  The general purpose profiles the WG will define will enable credential issuers to more easily build on existing IETF technologies.
+The SPICE WG will develop digital credential profiles which can support a number of use cases.
```
Comment (SteveLasker) [inline charter.md:31]: Should the below be a list, supporting the use cases?

Comment (OR13) [inline charter.md]: No because we don't want to focus on specific use cases in the charter.

**Review by SteveLasker — APPROVED** [review]
LGTM
Great to see SPICE progressing

**Review by mprorock — APPROVED** [review]

**Comment by mprorock** [comment]
Great work!

**Review by aj-stein-nist — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]
Awesome work

**Comment by mprorock** [comment]
@OR13  can we get this merged today?

File: charter.md (hunk 1 above)
Comment (henkbirkholz) [inline charter.md]:
```suggestion
The SPICE WG develops digital credential profiles which can support a number of use cases.
```

File: charter.md [hunk 2]
```diff
@@ -1,24 +1,20 @@
 ## Introduction
 
-Digital credentials are essential to identity, authorization, licenses, certificates and digitization use cases that are part of modernization efforts targeting efficiency, and transparency, such as digital licenses or certificates of origin.
+Digital credentials are essential to identity, authorization, licenses, certificates, and digitization use cases that are part of modernization efforts targeting efficiency and transparency.
 
-A digital credential expresses claims or attributes about a subject, such as their name or age, and their cryptographic keys.
-Some sets of claim names have already been defined by the IETF and other standards development groups (e.g., OpenID Foundation).
-Digital credentials typically involve at least three entities:
+A digital credential expresses claims or attributes about a subject, such as their name or age, and their cryptographic keys. Some sets of claim names have already been defined by the IETF and other standards development groups (e.g., OpenID Foundation).
```
Comment (henkbirkholz) [inline charter.md]:
```suggestion
A digital credential expresses claims, assertions, or attributes about a subject, such as their name or age, and their cryptographic keys. Some sets of claim names have already been defined by the IETF and other standards development groups (e.g., OpenID Foundation).
```

File: charter.md [hunk 3]
```diff
@@ -32,18 +28,29 @@ The SPICE WG will profile existing IETF technologies and address residual gaps t
...
 
-The SPICE WG will not develop digital credentials for any particular use case.  The general purpose profiles the WG will define will enable credential issuers to more easily build on existing IETF technologies.
+The SPICE WG develops digital credential profiles which can support a number of use cases.
+To help guide engineering decisions, requirements for proposed standards in the program of work will be created in coordination with the working groups listed above.
+The profiles developed by the SPICE WG will enable digital credentials to leverage existing IETF technologies.
+
+Privacy by design, confidentiality, and consent will be considered, and guidance will be given for each proposed standards in the program of work.
+
+The privacy and security considerations related to the impact of confidential computing, remote attestation, and hardware security module (HSM) on digital credentials will be developed in cooridination with relevant IETF WGs, and feedback from experts on the mailing list.
```
Comment (henkbirkholz) [inline charter.md]:
```suggestion
The privacy and security considerations related to the impact of confidential computing, remote attestation, and hardware security modules (HSM) on digital credentials will be developed in coordination with relevant IETF WGs, and feedback from experts on the mailing list.
```

**Review by henkbirkholz — APPROVED** [review]
This is now an awesome charter (I just ironed out a few nits).

:ship: it

**Pull request merged**